style/text-viz.less

```diff
@@ -14,6 +14,49 @@
         font: @text-value-size @text-value-font-family;
         color: @label-color;
         line-height: @text-value-line-height;
+        
+        span{
```

Comment:
Since we're not using CSS animations here, what if I set the duration to say 3-4 seconds, will it only show a single iteration of the animation (i.e. not loop)?